Formula/sundials.rb

```diff
@@ -24,7 +23,8 @@ class Sundials < Formula
   def install
     blas = "-L#{Formula["openblas"].opt_lib} -lopenblas"
     args = std_cmake_args + %W[
-      -DCMAKE_C_COMPILER=#{ENV["CC"]}
+      -DCMAKE_C_COMPILER=/usr/bin/clang
+      -DCMAKE_CXX_COMPILER=/usr/bin/clang++
```

Comment:
This bypasses the shims entirely during build which isn't what we want.

I argue that the makefiles should not be installed in the first place. They are generated build files - not reusable files for distribution.